src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicImmediate.cs

```diff
@@ -39,8 +39,8 @@ End Module
             VisualStudio.ImmediateWindow.ShowImmediateWindow(clearAll: true);
             VisualStudio.SendKeys.Send("?");
```

Comment:
VB unlike C# shows completion on ?

Reply:
@olegtk ? is a special thing, since VB makes a distinction between expressions and statements in the immediate window.